src/tcgwars/logic/impl/gen8/SwordShield.groovy

```diff
@@ -3417,7 +3417,7 @@ public enum SwordShield implements LogicCardInfo {
         def eff
         onPlay {reason->
           eff = delayed {
-            before (KNOCKOUT, self) {
+            before (KNOCKOUT, self) (priority: BEFORE_LAST) {
```

Comment:
priority param should go to `delayed` call

Reply:
Whoops.